.github/workflows/main.yml

```diff
@@ -91,7 +91,7 @@ jobs:
         if: ${{ !cancelled() }}
         shell: bash
   Lint:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why is this change in here?

Reply:
I guess this is a mistake, I don't think this PR is a proper place for such changes.